### DISCUSSION TITLE
Translate Output to Another Language

### DISCUSSION THREAD

**Discussion by SRoche14** [microsoft/pylance-release#3060] [Enhancement]
I was hoping it may be possible to translate the output of Pylance to another language, such as Spanish. The purpose of doing this would make it easier to understand the descriptions of functions and properties that Pylance shows. To accomplish this, my question is the following:

Is there already built-in support for this functionality, and if not, is there a way I can send the output of Pylance to a text file where I can use Python to translate the text myself? Ideally I would then be able to redirect the translated output to the normal channel Pylance is using. So essentially I would want to add an intermediate step before Pylance output is displayed.

Thanks!

**Comment by judej** [comment]
Moving this enhancement request (localization) to discussion as an enhancement request for comments and upvotes.
